docs/source/transformers/RenameVariables.rst

```diff
@@ -167,6 +167,19 @@ Allowed values are:
                 ${local}    Set Variable    value  # since we set it, ${local} is also known
                 Keyword Call    ${arg}    ${local}    ${global}  # ${global} is unknown
 
+Ignore variable case
+--------------------
+
+All variables case are converted according to configured conventions. It is possible to pass names of the variables
+that should be ignored. By default following variables case is ignored and not transformed:
+
+- ${\n}
```

Comment:
Is it a complete list?

Reply:
Yes. It could be longer if I decide to ignore True etc. But if not I will reword this part of the docs